### ISSUE TITLE
No notifications after recent update

### ISSUE THREAD

**Issue by VikingTechLLC** [sarahkittyy/KeywordTracker#30]
Just updated with the recent patch, 1.2.5, notice I am now not getting any notifications for any of my setup keywords/phrases.  
Refreshed, and restarted, still nothing.

**Comment by VikingTechLLC** [comment]
Tried uninstalling and reinstalling, get this compile error upon saving and trying to load
![Capture](https://user-images.githubusercontent.com/84103587/147510654-4106054b-9f99-4cb6-bbed-26e80970c208.PNG)

**Comment by VikingTechLLC** [comment]
Second time attempt
![image](https://user-images.githubusercontent.com/84103587/147510880-5d3f8539-d1a5-4ef9-8af4-7f03c35b04c4.png)

**Comment by sarahkittyy** [comment]
It works for me, I don't see any compilation errors.
Do you have the dependency? Where do you see this page "Addon Errors" because I haven't seen that tab lmao

**Comment by VikingTechLLC** [comment]
I tried multiple times over a 2 hour period, installing the prior release,
then updating through discord.  Also tried installing the proper package
directly downloading from the repository, each time it would just fail and
the page that came up was on discord, was a popup window that would show
only if I did a restart of discord.
I eventually was only able to get it to work by having a friend who also
uses the plugin send me the .js file and installing that one.
No clue why it worked that way, but it did first time with zero issues :/
